### DISCUSSION TITLE
import from ledger/hledger and similar plain text accounting formats

### DISCUSSION THREAD

**Discussion by simevo** [arrobalytics/django-ledger#342] [Ideas]
There are quite a few bookkeeping tools that use plain text files to store the ledger: https://plaintextaccounting.org

It would be useful to have scripts to import from those, in the first place the CoA, then transactions.

One could then import [ledger examples](https://github.com/simonmichael/hledger/tree/master/examples) and use them to create test cases:
- run hledger on the example -> get account balances
- import example ledger into django-ledger, run the reports -> compare balances with reference

Also interesting: import separate ledgers for each accounting period, consolidate them and get back the reports split over the accounting periods.

**Comment by elarroba** [comment]
@simevo thank you for requesting this feature... I think it would be great to support these open standards...
I will look in to it and add it to the road map... However, any pull requests are welcome to make the project move faster.
Thanks!
